src/libraries/Microsoft.PowerFx.Connectors/Tabular/Capabilities/ColumnCapabilitiesDefinition.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Text.Json.Serialization;
+using Microsoft.PowerFx.Connectors.Tabular.Capabilities;
+using Microsoft.PowerFx.Core.Utils;
+
+// DO NOT INCLUDE Microsoft.PowerFx.Core.Functions.Delegation.DelegationMetadata ASSEMBLY
+// as it defines CapabilitiesConstants which has invalid values.
```

Comment:
If there are errors in delegation, can we followup with Shantanu about this?

Reply:
Already done but no reply yet